test/resources/headerItems.html

```diff
@@ -0,0 +1 @@
ss="hidden"> news reports</span></a></li><li><a href="https://www.nhs.uk/news/pages/newsarticles.aspx?TopicId=Medication">Drugs and vaccines<span class="hidden"> news reports</span></a></li><li><a href="https://www.nhs.uk/news/pages/newsarticles.aspx?TopicId=Heart%2flungs">Heart and lungs<span class="hidden"> news reports</span></a></li><li><a href="https://www.nhs.uk/news/pages/newsarticles.aspx?TopicId=Medical+practice">Medical practice<span class="hidden"> news reports</span></a></li><li><a href="https://www.nhs.uk/news/pages/newsarticles.aspx?TopicId=Older+people">Older people and ageing<span class="hidden"> news reports</span></a></li><li><a href="https://www.nhs.uk/news/pages/newsarticles.aspx?TopicId=Genetics%2fstem+cells">Genetics and stem cells<span class="hidden">  news reports</span></a></li><li><a href="https://www.nhs.uk/news/pages/newsarticles.aspx?TopicId=Mental+health">Mental health<span class="hidden"> news reports</span></a></li><li><a href="https://www.nhs.uk/news/pages/newsarticles.aspx?TopicId=Diabetes">Diabetes<span class="hidden"> news reports</span></a></li><li><a href="https://www.nhs.uk/news">All Behind the Headlines news</a></li></ul></li><li><a href="https://www.nhs.uk/service-search">Services near you</a><ul><li><a href="https://www.nhs.uk/Service-Search/Accident-and-emergency-services/LocationSearch/428">A&amp;E<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Hospital/LocationSearch/7/Hospitals">Hospitals<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/GP/LocationSearch/4">GPs<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Dentist/LocationSearch/3">Dentists<span class="hidden">search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Pharmacy/LocationSearch/10">Pharmacies<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Care%20at%20home/LocationSearch/1833">Homecare and care homes<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Depression/LocationSearch/288">Depression<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Maternity-service/LocationSearch/1802">Maternity<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Alcohol-addiction/LocationSearch/1805">Alcohol<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Sexual-health-services/LocationSearch/1847">Sexual health<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Hospital/LocationSearch/7/Consultants">Consultants<span class="hidden"> search</span></a></li><li><a href="https://www.nhs.uk/Service-Search/Stop-smoking-services/LocationSearch/1846">Stop smoking services</a></li><li><a href="https://www.nhs.uk/Service-Search/Urgent-Care/LocationSearch/1824">Urgent care services</a></li></ul></li>
```

Comment:
Can this file end with a new line char or does that mess up the test? I assume this is the generated content? If its a problem don't worry about it.

Reply:
Adding a new line would mean also adding or removing a new line in the test assert, so best left as it is really.